src/core/stdc/math.d

```diff
@@ -3500,81 +3500,81 @@ else version(DragonFlyBSD)
     float ynf(int, float);
     float gammaf(float x);
     float dremf(float x, float);
-    int finitef(float x);
-    int isinff(float x);
-    int isnanf(float x);
+    pure int finitef(float x);
+    pure int isinff(float x);
+    pure int isnanf(float x);
 
     float gammaf_r(float x, int *);
     float lgammaf_r(float x, int *);
     float significandf(float x);
 
     /* real */
-    real acosl(real x);
-    real asinl(real x);
-    real atanl(real x);
-    real atan2l(real x, real y);
-    real cosl(real x);
-    real sinl(real x);
-    real tanl(real x);
-
-    real acoshl(real x);
-    real asinhl(real x);
-    real atanhl(real x);
-    real coshl(real x);
-    real sinhl(real x);
-    real tanhl(real x);
-
-    real expl(real x);
-    real exp2l(real x);
-    real expm1l(real x);
-    real frexpl(real x, int *exp);
+    pure real acosl(real x);
+    pure real asinl(real x);
+    pure real atanl(real x);
+    pure real atan2l(real x, real y);
```

Comment:
For non-DragonFlyBSD `atan2` is not marked `pure`

Reply:
Thanks for checking all of these. I guess i thought i saw a pattern when making the corrections, which was faulty.